lib/share_progress/email_variant.rb

```diff
@@ -0,0 +1,22 @@
+require 'share_progress'
+require 'share_progress/variant'
+
+module ShareProgress
+  class EmailVariant < Variant
+    attr_accessor :email_subject, :email_body
```

Comment:
If we've got `fields` available here, could this be changed to `attr_accessor EmailVariant.fields`?